flowzone.yml

```
@@ -3312,6 +3360,86 @@ jobs:
 
       - *waitForCloudFormation
 
+  ###################################################
+  # Terraform
+  ###################################################
+
+  terraform_test:
+    name: Test Terraform
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    strategy:
+      fail-fast: true
+      matrix:
+        project: ${{ fromJSON(needs.is_terraform.outputs.projects) }}
```

```suggestion
        project: ${{ fromJSON(needs.is_terraform.outputs.deploy) }}
```
You've named your output `deploy` on line 1822